### PR TITLE
Tokens Management

### PR DESCRIPTION
**Pull request by Kakulukian**

### CODE REVIEW

File: docs/hub/enterprise-hub-tokens-management.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Tokens Management allows organization administrators to control access tokens within their organization, ensuring that only authorized users have access to organization resources.
+
+
+## Viewing and Managing Access Tokens
+
+The token listing feature provides a view of all access tokens within your organization. Administrators can:
+
+* Monitor token usage and identify potential security risks
+* Identify inactive or unused tokens that can be revoked
```
Comment (julien-c) [inline docs/hub/enterprise-hub-tokens-management.md:15]:
```suggestion
- Monitor token usage and identify or prevent potential security risks:
  - unauthorized access to private resources ("leak")
  - scopes of access that are too wide
  - improveable token hygienics (tokens that have not been rotated in a long time, etc) 
- Identify inactive or unused tokens that can be revoked
```

maybe list a few potential, concrete risks that make this a very useful feature for companies

**Review by julien-c — APPROVED** [review]
very nice!

File: docs/hub/enterprise-hub-tokens-management.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+<div class="flex justify-center">
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/tokens-management-policy.png" />
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/tokens-management-policy.png" />
+</div>
+
+
+## Reviewing Token Authorization
+
+If your token policy requires an approval before accessing your resources, organization administrators can view all fine-grained tokens that can access resources owned by the organization. Organization owners can also revoke access by fine-grained personal access tokens. Organization administrators will receive an email when an authorization is requested.
```
Comment (Pierrci) [inline docs/hub/enterprise-hub-tokens-management.md:52]:
```suggestion
When your token policy is set to "Require administrator approval", organization administrators can view the details of all fine-grained tokens with access to resources owned by the organization. They can also revoke access to those tokens. Organization administrators will receive an email when an authorization is requested for a fine-grained token.
```

File: docs/hub/enterprise-hub-tokens-management.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+When a token is revoked or denied, the user who created the token will receive an email notification.
+
+Organization owners can only view and revoke fine-grained personal access tokens, not personal access tokens.
+
+
```
Comment (Pierrci) [inline docs/hub/enterprise-hub-tokens-management.md:58]:
```suggestion
```
I would just remove this, I don't think it's needed and the wording is confusing.

**Review by Pierrci — APPROVED** [review]
Nice, I left a few suggestions, but lgtm! 👍

File: docs/hub/enterprise-hub-tokens-management.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+## Viewing and Managing Access Tokens
+
+The token listing feature provides a view of all access tokens within your organization. Administrators can:
+
+- Monitor token usage and identify or prevent potential security risks:
+  - unauthorized access to private resources ("leak")
+  - scopes of access that are too wide
+  - improveable token hygienics (tokens that have not been rotated in a long time, etc)
```
Comment (pcuenca) [inline docs/hub/enterprise-hub-tokens-management.md:17]:
```suggestion
  - improvable token hygienics (tokens that have not been rotated in a long time, for example) 
```

or maybe just `improved`

File: docs/hub/enterprise-hub-tokens-management.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  - improveable token hygienics (tokens that have not been rotated in a long time, etc) 
+- Identify inactive or unused tokens that can be revoked
+
+<div class="flex justify-center">
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/tokens-management-list.png" />
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/tokens-management-list-dark.png" />
+</div>
+
+
+Each fine-grained token can be reviewed to see their permissions
```
Comment (pcuenca) [inline docs/hub/enterprise-hub-tokens-management.md:26]:
```suggestion
Fine-grained tokens can be reviewed to see their permissions:
```

File: docs/hub/organizations-security.md [hunk 6]
```diff
@@ -49,3 +49,7 @@ You may be able to view the email addresses of members of your organization. The
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/org-members-page-emails-dark.png"/>
 </div>
 
+
+# Managing Access Tokens with access to my organization
```
Comment (pcuenca) [inline docs/hub/organizations-security.md:53]: should this be a `##`?

Comment (Kakulukian) [inline docs/hub/organizations-security.md:53]:
```suggestion
## Managing Access Tokens with access to my organization
```

File: docs/hub/enterprise-hub-tokens-management.md (hunk 3 above)
Comment (julien-c) [reply docs/hub/enterprise-hub-tokens-management.md]: hmm are you sure? i thought it was useful info

Comment (Pierrci) [reply docs/hub/enterprise-hub-tokens-management.md]: it was a bit redundant with the paragraph above IMO, and `fine-grained personal access tokens` vs `personal access tokens` is very confusing for me

**Pull request merged**